rel/overlay/etc/vm.args

```diff
@@ -60,3 +60,22 @@
 
 # Set maximum SSL session lifetime to reap terminated replication readers
 -ssl session_lifetime 300
+
+## TLS distribution
+## Use TLS for connections between Erlang cluster members.
+## http://erlang.org/doc/apps/ssl/ssl_distribution.html
+## Don't forget to override the paths to point to your certificate(s) and key(s)!
+## Once a connection is established, Erlang doesn't differentiate between
+## a client and a server - the same certs/keys can be used on both sides.
+#-proto_dist inet_tls
```

Comment:
this should be `couch`. I believe there is a convention imposed by Erlang OTP that the `proto_dist couch` would be calling `couch_dist` module. Therefore you would also need to rename `couch_ssl` into `couch_dist`.

Reply:
Change to `couch_dist`, thanks for the review.